Why: WebAssembly support for libffi
Source: Ported by Kleis Auke Wolthuizen (wasm-vips), based on https://github.com/brion/libffi/commits/emscripten-work and https://github.com/hoodmane/libffi-emscripten

diff --git a/src/closures.c b/src/closures.c
index 9aafbec4b8c2a10ad77158fd5d58e46e17398a08..58c2c21da020e93d9429dc346de5ab6a74a2f036 100644
--- a/src/closures.c
+++ b/src/closures.c
@@ -31,6 +31,8 @@
 #define _GNU_SOURCE 1
 #endif
 
+#ifndef __EMSCRIPTEN__
+
 #include <fficonfig.h>
 #include <ffi.h>
 #include <ffi_common.h>
@@ -1098,3 +1100,4 @@ ffi_tramp_is_present (__attribute__((unused)) void *ptr)
 #endif /* FFI_CLOSURES */
 
 #endif /* NetBSD with PROT_MPROTECT */
+#endif /* __EMSCRIPTEN__ */
